src/lib/tabs/Tabs.tsx

```diff
@@ -5,6 +5,7 @@ import { Tabs as MuiTabs, Tab } from '@material-ui/core';
 
 interface TabPanelProps {
   children?: React.ReactNode;
+  className: any,
```

Comment:
children and className shouldn't be needed here.

`React.FC<TabPanelProps>` React.FC stands for React Functional Component and adds all React standard props already.

Reply:
This way throwing a TS error without having it defined inside of the interface, that error was the reason the first build for `0.3.0` failed. Are you sure it isn't necessary?

Reply:
I'll check her out on my end

Reply:
Just checked, here's the error being thrown by TS:
```[!] (plugin typescript) Error: @rollup/plugin-typescript TS2322: Type '{ children: ReactNode; className: string; value: number; index: number; }' is not assignable to type 'IntrinsicAttributes & TabPanelProps & { children?: ReactNode; }'.
  Property 'className' does not exist on type 'IntrinsicAttributes & TabPanelProps & { children?: ReactNode; }'.```

For whatever reason, it looks like the `<TabPanel>` component doesn't have access to `className` without it being defined in the interface.

Reply:
oh className was needed, but not children. I'm actually getting rid of the need for any of these styles I think.